src/main/java/com/dislinkt/post/model/Comment.java

```diff
@@ -6,7 +6,6 @@
 public class Comment {
     
     @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Integer id;
```

Comment:
Why did you remove this?

Reply:
For some reason, it wouldn't allow me to create new entities unless the database was empty upon start.

Reply:
Hmm, ok, lets see about that in next ticket.